app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
-  [ ] Hello @arslanbisharat, we need to implement the models correctly according to the diagram.  Nice work on your schema.rb, but the Model should be configured too according to the requirements:

![image](https://user-images.githubusercontent.com/57421823/172322667-84bf8ef4-ad38-483a-a962-3ef10e0718cf.png)

This is an example:

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
@Meltrust , thanks for clarifying. This helped a lot.